quickstart/aspnet-core/azure.yaml

```diff
@@ -0,0 +1,37 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-dev/main/schemas/v1.0/azure.yaml.json
```

Comment:
Why do we have so many commented lines in this file? Do we need them?

Reply:
fixed